package/cloudshell/iac/terraform/tagging/tag_terraform_resources.py

```diff
@@ -732,8 +731,59 @@ def _validate_terraform_version_arg(terraform_version_arg: str) -> bool:
     if not terraform_version_arg:
         return False
 
-    version_arr = terraform_version.split(".")
+    version_arr = terraform_version_arg.split(".")
     if len(version_arr) != 3:
         return False
 
     return version_arr[0].isdigit() and version_arr[1].isdigit()
+
+
+# def _try_get_settings() -> Settings:
+#     parser = argparse.ArgumentParser()
+#
+#     # mandatory positional arguments
+#     parser.add_argument('tf_version',
+#                         help="TF version, e.g. 0.10.0, 0.12.0, 0.15.0, ...")
+#     parser.add_argument('tf_module_dir_path',
+#                         help="path to the folder with the tf files")
+#     parser.add_argument('tags_file_path',
+#                         help="path to the json file with the tags to be applied to the module")
+#
+#     # optional named arguments
+#     parser.add_argument('--tf_exe_path', default="",
+#                         help="path to the TF executable, e.g. /usr/bin/terraform. "
+#                              "if not specified, PATH locations will be searched for 'terraform' exe")
+#     parser.add_argument('--tf_vars_file_path', default="",
+#                         help="path to the vars file for the module")
+#     parser.add_argument('--exclude_from_tagging_file_path', default="",
+#                         help="path to the json file with the resources to be excluded from tagging")
+#
+#     settings = Settings()
+#     # if invalid, will print error + usage and exit
+#     parser.parse_args(namespace=settings)
+#
+#     # additional validations
+#     if settings.tf_exe_path and not os.path.isfile(settings.tf_exe_path):
+#         parser.print_usage()
+#         print("error: tf_exe_path value is invalid")
+#         exit(1)
+#
+#     if not _validate_terraform_version_arg(settings.tf_version):
+#         parser.print_usage()
+#         print("error: tf_version value is invalid")
```

Comment:
same thing, should we just remove all this now?

Reply:
done